lib/usecases/log/GetAllLogsUseCase.js

```diff
@@ -169,6 +176,29 @@ class GetAllLogsUseCase {
                 queryBuilder.where('id').oneOf(...logIds);
             }
 
+            if (filter && filter.environments && filter.environments.values.length > 0) {
```

Comment:
You can use `?.` here (on backend only for now)
`filter?.environment?.values?.length > 0`